### DISCUSSION TITLE
Alternative software

### DISCUSSION THREAD

**Discussion by Rynothefox** [upscayl/upscayl#368] [Q&A] [answered]
Hey folks just wanted to say Upscayl is so good!
Thank you for this software!
I find sometimes images need sharpening and have been using this website first:
https://picwish.com/photo-enhancer to bring up the quality, then upscale.

Q: Can anyone her make recommendations for free open source image enhancements, like unblurring photos? Ta
 Please and thank you!

**Comment by aaronliu0130** [comment, marked as answer]
Enhancements are filed in the [upscayl repository's issues](https://github.com/upscayl/upscayl/issues). Unblurring is already at https://github.com/upscayl/upscayl/issues/275.
